src/test/java/uk/gov/ida/verifylocalmatchingserviceexample/service/Cycle1MatchingServiceTest.java

```diff
@@ -11,218 +14,180 @@
 
-        when(personDAO.getMatchingUsers(surnames, dateOfBirth)).thenReturn(matchingUsers);
+    @Before
+    public void setUp() {
+        personDAO = mock(PersonDAO.class);
+        verifiedPidDAO = mock(VerifiedPidDAO.class);
+        matchingServiceRequestDto = mock(MatchingServiceRequestDto.class, Answers.RETURNS_DEEP_STUBS);
 
         cycle1MatchingService = new Cycle1MatchingService(personDAO, verifiedPidDAO);
-        MatchingServiceRequestDto requestWithVerifiedSurnameAndDateOfBirth = aMatchingServiceRequestDtoBuilder()
-                .withMatchingAttributesDto(aMatchingAttributesDtoBuilder()
-                        .withSurname(getVerifiedSurname(surnames.getFirst()))
-                        .withDateOfBirth(verifiedDateOfBirth)
-                        .build())
-                .build();
-
-        MatchStatusResponseDto matchStatusResponseDto = cycle1MatchingService.matchUser(requestWithVerifiedSurnameAndDateOfBirth);
 
-        verify(personDAO, times(1)).getMatchingUsers(surnames, dateOfBirth);
-        assertThat(matchStatusResponseDto).isEqualTo(MatchStatusResponseDto.MATCH);
+        anyValidatedDateOfBirth = aMatchingAttributesValueDtoBuilder().withValue(LocalDate.now()).withVerified(true).build();
+        anyListOfSurnamesWithValidatedValue = Arrays.asList(
+            aMatchingAttributesValueDtoBuilder().withValue("any-surname-one").withVerified(true).build(),
+            aMatchingAttributesValueDtoBuilder().withValue("any-surname-two").withVerified(true).build(),
+            aMatchingAttributesValueDtoBuilder().withValue("any-surname-three").withVerified(false).build()
+        );
     }
 
     @Test
-    public void shouldReturnNoMatchWhenUserWithGivenSurnameAndDateOfBirthNotFoundInTheDatabase() {
-        LinkedList<String> surnames = new LinkedList<String>() {{
-            add("test-surname");
-        }};
-        LocalDate dateOfBirth = LocalDate.of(1993, 12, 16);
-        MatchingAttributesValueDto verifiedDateOfBirth = getVerifiedDateOfBirth(dateOfBirth);
-        when(personDAO.getMatchingUsers(surnames, dateOfBirth)).thenReturn(null);
-        cycle1MatchingService = new Cycle1MatchingService(personDAO, verifiedPidDAO);
-
-        MatchingServiceRequestDto requestWithVerifiedSurname = aMatchingServiceRequestDtoBuilder()
-                .withMatchingAttributesDto(aMatchingAttributesDtoBuilder()
-                        .withSurname(getVerifiedSurname(surnames.getFirst()))
-                        .withDateOfBirth(verifiedDateOfBirth)
-                        .build())
-                .build();
+    public void shouldReturnNoMatchWhenDateOfBirthIsNotVerified() {
+        MatchingAttributesValueDto<LocalDate> dateOfBirth = matchingServiceRequestDto.getMatchingAttributesDto().getDateOfBirth();
 
-        MatchStatusResponseDto matchStatusResponseDto = cycle1MatchingService.matchUser(requestWithVerifiedSurname);
+        when(dateOfBirth.getVerified()).thenReturn(false);
```

Comment:
DdateOfBirth is not mocked. It's actual object.

Reply:
DateOfBirth is mocked. The code is correct. However the test passed even with reversed value because a verify was missing.